includes/sanitizers/class-amp-base-sanitizer.php

```diff
@@ -304,4 +304,46 @@ public function maybe_enforce_https_src( $src, $force_https = false ) {
 
 		return $src;
 	}
+
+	/**
+	 * Removes a child of a node.
+	 *
+	 * Also, calls the mutation callback for it.
+	 * This tracks all the nodes that were removed.
+	 *
+	 * @since 0.7
+	 *
+	 * @param DOMNode $child The node to remove.
+	 * @return void.
+	 */
+	public function remove_child( $child ) {
+		if ( method_exists( $child->parentNode, 'removeChild' ) ) { // phpcs:ignore WordPress.NamingConventions.ValidVariableName.NotSnakeCaseMemberVar.
+			$child->parentNode->removeChild( $child ); // phpcs:ignore WordPress.NamingConventions.ValidVariableName.NotSnakeCaseMemberVar.
+			if ( isset( $this->args['mutation_callback'] ) ) {
+				call_user_func( $this->args['mutation_callback'], $child, 'removed' );
```

Comment:
I believe `AMP_Mutation_Util::NODE_REMOVED` could be used here.

Reply:
Thanks, this [commit](https://github.com/Automattic/amp-wp/commit/69317b877679ceb4bfb4a319244de0995f1b1222) uses that constant instead of the string.